.github/workflows/tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Run weaver tests
+
+on:
+  push:
+  workflow_dispatch:
```

Comment:
Don't think this is valid YAML, since the workflow didn't trigger on this PR. I think you want:
```suggestion
on: [push, workflow_dispatch]
```
unless there's more configs you wanted to set on each of them.

Reply:
From what I recall, the workflow won't trigger until the code is first merged onto the default branch. It ran on my fork once I merged it into `master`: https://github.com/plocket/docassemble-assemblylinewizard/actions/runs/780967867

Reply:
Weird. I've definitely gotten workflows to run on non-default branches, but it's probably some weird combination of ownership and other obtuse settings. As long as it works. :+1:

Reply:
Actually, it does run and is running! I guess they changed that at some point. Awesome! Not sure why it didn't initially get triggered then.